package.json

```diff
@@ -29,12 +29,13 @@
     "@nestjs/platform-express": "^10.0.0",
     "@nestjs/swagger": "^7.1.13",
     "@nestjs/typeorm": "^10.0.0",
+    "@supabase/supabase-js": "^2.39.0",
     "cache-manager": "^5.2.3",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
     "helmet": "^7.0.0",
     "langchain": "^0.0.174",
-    "openai": "^4.14.1",
+    "openai": "^4.20.0",
```

Comment:
Needed to upgrade because openai version 4.14 doesn't have the [new assistants API](https://platform.openai.com/docs/assistants/overview), which is needed for hash-openai